modules/administration/pages/tshoot-registerclones.adoc

```diff
@@ -1,186 +1,128 @@
 [[troubleshooting-register-clones]]
 = Troubleshooting Registering Cloned Minions
 
+////
+PUT THIS COMMENT AT THE TOP OF TROUBLESHOOTING SECTIONS
 
+Troubleshooting format:
 
+One sentence each:
+Cause: What created the problem?
+Consequence: What does the user see when this happens?
+Fix: What can the user do to fix this problem?
+Result: What happens after the user has completed the fix?
 
+If more detailed instructions are required, put them in a "Resolving" procedure:
+.Procedure: Resolving Widget Wobbles
+. First step
+. Another step
+. Last step
+////
 
+Sometimes a cloned client (either traditional or Salt) will use the same machine ID as the system they are a clone of.
+This results in {productname} only recognising one system, rather than two different systems.
+This can be resolved by changing the machine ID of the  cloned system, so that {productnamne} recognises them as two different clients.
```

Comment:
```suggestion
This can be resolved by changing the machine ID of the  cloned system, so that {productnamne} recognizes them as two different clients.
```

Reply:
:) Nice catch Karl